admin/admin.go

```diff
@@ -329,6 +329,19 @@ func getOffsetResult(res *sarama.OffsetResponse, topic string, partition int32)
 	return block.Offsets[0], nil
 }
 
+func (a *T) GetTopics() ([]string, error) {
+	kafkaClt, err := a.lazyKafkaClt()
+	if err != nil {
+		return nil, err
+	}
+
+	topics, err := kafkaClt.Topics()
```

Comment:
Is it possible to retrieve topic parameters such as number of partitions, replication factor, retention period, etc. as well? It would be nice to return that info.

Reply:
I added some parameters. Please see below.

Reply:
I added some test cases for that.

Reply:
`meta` and `cfg` sounds like the same thing to me, I had to read the code to understand what the difference is. To avoid this confusion I suggest renaming the parameters to `withPartitions` and `withConfig` respectively.

Reply:
Addressed